pytorch_toolkit/tests/run_model_templates_tests.py

```diff
@@ -94,6 +94,39 @@ def test_existence_of_mandatory_files_in_template_dir(self):
             self.assertTrue(os.path.exists(os.path.join(template_dirname, 'requirements.txt')))
             self.assertTrue(os.path.exists(os.path.join(template_dirname, 'modules.yaml')))
 
+    def test_existence_of_mandatory_fields_in_template(self):
+        mandatory_fields = [
+            'name',
+            'domain',
+            'problem',
+            'framework',
+            'summary',
+            'annotation_format',
+            'initial_weights',
+            'dependencies',
+            'max_nodes',
+            'training_target',
+            'inference_target',
+            'hyper_parameters',
+            'output_format',
+            'optimisations',
+            'metrics',
+            'tensorboard',
+        ]
+        template_files = self._get_template_files()
+        for template in template_files:
+            with open(template) as read_file:
+                template_fields = list(yaml.load(read_file, yaml.SafeLoader).keys())
+            max_pos = -1
+            for field in mandatory_fields:
```

Comment:
@Ilya-Krylov :  
Ilya, what do you think on the more pythonic way?
It is not mandatory, but I could suggest something like
```python
if not set(template_fields) >= set(mandatory_fields):
    absent_fields = set(mandatory_fields) - set(template_fields)
    raise RuntimeError(f'Template {template} does not contain mandatory fields {absent_fields}')

template_mandatory_fields = [s for s in template_fields if s in mandatory_fields]
if template_mandatory_fields != mandatory_fields:
    diff_set = set(enumerate(template_mandatory_fields)) - set(enumerate(mandatory_fields))
    first_diff_field = sorted(diff_set)[0][1]
    raise RuntimeError(f'Wrong order of mandatory fields in template {template} from the field {first_diff_field}')
```

Reply:
Thank you for your suggestion, but I agree that this is not mandatory, and my code is simpler I guess though it is no so pythonic as yours 😉

Reply:
Ok!